driver-mct/cime_config/config_component_e3sm.xml

```diff
@@ -725,7 +726,9 @@
       <value compset="^2000.+AV1C-H01C"				>368.865</value>
       <value compset="^1850.+CMIP6"                             >284.317</value>
       <value compset="^1850.+CMIP6.+4xCO2"			>1137.268</value>
+      <value compset="^1950"                                    >312.821</value>
       <value compset="^1950.+CMIP6"                             >312.821</value>
+      <value compset="^2010"                                    >388.717</value>
```

Comment:
It will match any compset that starts with `2010`. F2010-SCREAMv1 is just an alias for a much longer full name that does start with 2010:

```
  <compset>
      <alias>F2010-SCREAMv1</alias> <!-- Coupled land-atm compset (using SCREAMv1), 2010 initial conditions -->
      <lname>2010_SCREAM_ELM%SPBC_CICE%PRES_DOCN%DOM_SROF_SGLC_SWAV_SIAC_SESP</lname>
      <support_level>Experimental, under development</support_level>
  </compset>
```

Reply:
Understandable. I think `<values match="last">` means go down the list of all these values and take the last one that matches. For `^2010`, this is regex syntax and `^` is a special character that refers to the start of the string.

Reply:
Side topic: I was thinking about `^` and I don't understand how `F2010` is a match, given that `2010` is not at the start of the string...

Reply:
Nvm, @mahf708 reminded me about longnames.